pkg/hyperkit/driver.go

```diff
@@ -404,3 +409,14 @@ func (d *Driver) getPid() int {
 
 	return config.Pid
 }
+func (d *Driver) UpdateConfigRaw(rawConfig []byte) error {
```

Comment:
This shouldn't be part of https://github.com/code-ready/machine-driver-hyperkit/pull/19/commits/482ba186d3323b5130f26e1b76829063bdce7294 commit.